skimage/transform/radon_transform.py

```diff
@@ -18,23 +19,27 @@
 __all__ = ['radon', 'order_angles_golden_ratio', 'iradon', 'iradon_sart']
 
 
-def radon(image, theta=None, circle=True):
+def radon(image, theta=None, circle=True, preserve_range=None):
```

Comment:
```suggestion
def radon(image, theta=None, circle=True, *, preserve_range=None):
```

These kind of "behavior" parameters should definitely be made keyword only (something that wasn't possible in python 2.7)

Reply:
Done